### PR TITLE
 Added a lessons count/total

### PR DESCRIPTION
**Pull request by AdnCodez**
This PR closes the issue #55

### CODE REVIEW

File: public/js/index.js [hunk 1]
```diff
@@ -30,6 +30,7 @@ const lessonInput = document.querySelector(".create-lesson-input");
 const formElement = document.querySelector("form");
 const submitLessonElement = document.querySelector("#submit");
 const lessonsContainer = document.querySelector(".lessons");
+const LessonCount = document.querySelector(".lessons-count");
```
Comment (jrobind) [inline public/js/index.js]: In JS it's convention to only capitalise Classes and constructor functions. Could you change this to `lessonCount` please.

**Review by jrobind — CHANGES_REQUESTED** [review]
Looks good - thanks! Just one minor comment to address and then it's ready to go 😄

File: public/js/index.js (hunk 1 above)
Comment (AdnCodez) [reply public/js/index.js]: oh yeah didn't catch that, I will

**Comment by jrobind** [comment]
Looks good – thanks!

**Pull request merged**